TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/dialog/NoticeDialog.java

```diff
@@ -0,0 +1,75 @@
+package com.ybsystem.tweetmate.fragments.dialog;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.TextView;
+
+import androidx.fragment.app.DialogFragment;
+
+import com.ybsystem.tweetmate.R;
+import com.ybsystem.tweetmate.utils.CalcUtils;
+
+import static com.ybsystem.tweetmate.resources.ResString.*;
+
+public class NoticeDialog extends DialogFragment {
+
+    private DialogInterface.OnClickListener mPositiveClickListener;
+
+    public NoticeDialog newInstance(String description) {
+        Bundle bundle = new Bundle();
+        bundle.putString("DESCRIPTION", description);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "DESCRIPTION" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1192)